crates/bindings-csharp/BSATN.Runtime/Builtins.cs

```diff
@@ -349,8 +349,8 @@ public AlgebraicType GetAlgebraicType(ITypeRegistrar registrar) =>
             // to avoid leaking the internal *Repr wrappers in generated SATS.
             new AlgebraicType.Sum(
                 [
-                    new("Time", new AlgebraicType.U64(default)),
                     new("Interval", new AlgebraicType.U64(default)),
+                    new("Time", new AlgebraicType.U64(default)),
```

Comment:
FWIW this is the only necessary reordering, the rest don't matter and can be safely reverted.

Reply:
I figured, but I thought we might as well update all of them for consistency.